chat/src/main/java/ChatManagement.java

```diff
@@ -0,0 +1,22 @@
+import static com.codeborne.selenide.Selectors.byText;

```

Comment:
This file must be located in a package 
`package org.exoplatform.platform.qa.ui.chat.pageobject;`